### ISSUE TITLE
[DISCUSS]Whether the creation of cluster_schema database is allowed in the new tenant

### ISSUE THREAD

**Issue by Benxiaohai001** [cnosdb/cnosdb#1567]
### Describe the bug

Whether the creation of cluster_schema database is allowed in the new tenant.The current situation is that when creating a cluster_schema database in a new tenant, an error is reported: cluster_schema already exists.In fact, we only created the cluster_schema database under the cnosdb database. It does not exist under the new tenant.When we switch to the cluster_schema database and create a table, another error will be reported: cluster_schema not found.This situation is inconsistent, and I hope there can be a better way to handle it.
![image](https://github.com/cnosdb/cnosdb/assets/40594937/67879629-532c-4c0d-a384-e1596421d838)

**Comment by Benxiaohai001** [comment]
```log
public ❯ create database cluster_schema;
Query took 0.010 seconds.
public ❯ \c cluster_schema
cluster_schema ❯ \d
Query took 0.004 seconds.
cluster_schema ❯ CREATE TABLE air (visibility DOUBLE,temperature DOUBLE,pressure DOUBLE,TAGS(station));
INSERT INTO air (TIME, station, visibility, temperature, pressure) VALUES (now(), 'XiaoMaiDao1', 56, 69, 411);
Query took 0.008 seconds.
+------+
| rows |
+------+
| 1    |
+------+
Query took 0.093 seconds.
cluster_schema ❯ \d
+------------+
| table_name |
+------------+
| air        |
+------------+
Query took 0.005 seconds.
```
version: cnosdb 2.4.0, revision db434d4696e0c58e6881cc89e49cabece0316bff

**Comment by Benxiaohai001** [comment]
@roseboy-liu The PR fix this issue need sync LTS/2.3.

**Comment by Benxiaohai001** [comment]
```log
public ❯ create database cluster_schema;
Query took 0.005 seconds.
public ❯ \c cluster_schema
cluster_schema ❯ \d
Query took 0.009 seconds.
cluster_schema ❯ CREATE TABLE air (visibility DOUBLE,temperature DOUBLE,pressure DOUBLE,TAGS(station));
INSERT INTO air (TIME, station, visibility, temperature, pressure) VALUES (now(), 'XiaoMaiDao1', 56, 69, 411);

Query took 0.002 seconds.
+------+
| rows |
+------+
| 1    |
+------+
Query took 0.016 seconds.
cluster_schema ❯ 
```
version: cnosdb 2.3.4, revision 07e229d4081c61b5012f70c3254e534ffe05d2e9
